remix-analyzer/src/solidity-analyzer/modules/abstractAstView.ts

```diff
@@ -46,23 +46,23 @@ export default class abstractAstView {
  * @contractsOut {list} return list for high level AST view
  * @return {ASTNode -> void} returns a function that can be used as visit function for static analysis modules, to build up a higher level AST view for further analysis.
  */
-  build_visit (relevantNodeFilter) {
-    var that = this
-    return function (node) {
-      if (isContractDefinition(node)) {
+  build_visit (relevantNodeFilter: ((node:any) => boolean)): VisitFunction {
+    const that: abstractAstView = this
+    return function (node: any) {
+      if (node.nodeType === "ContractDefinition") {
         that.setCurrentContract(that, {
           node: node,
           functions: [],
           relevantNodes: [],
           modifiers: [],
           inheritsFrom: [],
-          stateVariables: getStateVariableDeclarationsFormContractNode(node)
+          stateVariables: getStateVariableDeclarationsFromContractNode(node)
         })
-      } else if (isInheritanceSpecifier(node)) {
-        const currentContract = that.getCurrentContract(that)
-        const inheritsFromName = getInheritsFromName(node)
+      } else if (node.nodeType === "InheritanceSpecifier") {
+        const currentContract: ContractHLAst = that.getCurrentContract(that)
+        const inheritsFromName: string = getInheritsFromName(node)
```

Comment:
The method `getInheritsFromName` shld be defined to return a string so you don't have to hardtype it.